client/src/infographic/Infographic.js

```diff
@@ -218,84 +229,91 @@ class InfoGraph_ extends React.Component {
     );
 
     return (
-      <div>
-        <AnalyticsSynchronizer {...this.props} />
-        {window.is_a11y_mode && (
+      <InfographicContext.Provider
+        value={{
+          get_copy_link: this.get_copy_link,
+          get_link_state: this.get_link_state,
+        }}
+      >
```

Comment:
I just wanna double check, none of the code below was actually changed right?

Reply:
I don't think i changed anything other than the context